frontend/src/pages/intake-form/fields.ts

```diff
@@ -1,38 +1,279 @@
-import { Program } from '@/common';
+// common
+import { BcwsRoleInterface, Languages, PersonnelTool, Program } from '@/common';
 
 export interface ProgramFields {
   program?: Program;
 }
-// program selection
+// core team program selection
 export const programFields: ProgramFields = {
   program: undefined,
 };
 
-export interface PersonalInfoFields {
+// acknowledgement for selected program stream(s)
+export interface AcknowledgementEmcr {
+  expectations: boolean;
+  approvedBySupervisor: boolean;
+}
+
+export type AcknowledgementBcws = AcknowledgementEmcr & {
+  orientation: boolean;
+  willingnessStatement: boolean;
+  parQ: boolean;
+};
+
+export type AcknowledgementBoth = AcknowledgementEmcr & AcknowledgementBcws;
+
+export const acknowledgementEmcr: AcknowledgementEmcr = {
+  expectations: false,
+  approvedBySupervisor: false,
+};
+
+export const acknowledgementBcws: AcknowledgementBcws = {
+  expectations: false,
+  approvedBySupervisor: false,
+  orientation: false,
+  willingnessStatement: false,
+  parQ: false,
+};
+
+export const acknowledgementBoth: AcknowledgementBoth = {
+  expectations: false,
+  approvedBySupervisor: false,
+  orientation: false,
+  willingnessStatement: false,
+  parQ: false,
+};
+
+// personal & employee information
+
+// personal details
+
+export interface PersonalDetails {
   firstName: string;
   lastName: string;
-  email: string;
-  // ...etc
+  primaryPhone: string;
+  secondaryPhone?: string;
+  homeLocation: string;
```

Comment:
home location will be an object containing a fire centre and region as well, I believe.